pumpkin-lib/src/propagators/cumulative/time_table/time_table_over_interval_incremental.rs

```diff
@@ -63,15 +66,20 @@ pub(crate) struct TimeTableOverIntervalIncrementalPropagator<Var> {
     time_table: OverIntervalTimeTableType<Var>,
     /// Stores the input parameters to the cumulative constraint
     parameters: CumulativeParameters<Var>,
-    /// Keeps track of whether the current state of the
-    /// [`TimeTableOverIntervalIncrementalPropagator::time_table`] is outdated (i.e. whether it
-    /// needs to be recalculated from scratch). This can occur due to backtracking.
+    /// Stores structures which change during the search; either to store bounds or when applying
+    /// incrementality
+    dynamic_structures: DynamicStructures<Var>,
```

Comment:
The name `dynamic_structures` is quite ambiguous. This applies to the field name and the type name

Reply:
I would agree but I cannot think of a clearer name for it, I have changed it to `UpdatableStructures` but please let me know if you have any better names in mind!